internal/manifest/os.go

```diff
@@ -140,7 +131,7 @@ func (p *OS) getPackageSetChain() []rpmmd.PackageSet {
 	}
 
 	// If we have a logical volume we need to include the lvm2 package
-	if p.PartitionTable != nil && p.OSTree == nil {
+	if p.PartitionTable != nil && p.OSTreeParent == nil {
```

Comment:
Oh, good catch.  That's actually not the reason.  The reason we were checking `OSTree` is because partition tables don't make sense when building an ostree commit or container.  `p.OSTree` got renamed to `p.OSTreeParent` but that's optional and in fact we should be checking `p.OSTreeRef`, which is the way of detecting if we're building an ostree type.